src/coreclr/src/libraries-native/CMakeLists.txt

```diff
@@ -1,11 +1,22 @@
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
-set(GLOBALIZATION_NATIVE_DIR ${CLR_REPO_ROOT_DIR}/src/libraries/Native/Unix/System.Globalization.Native)
 
 # Suppress exporting of the PAL APIs
 add_definitions(-DPALEXPORT=EXTERN_C)
 
-include_directories("${GLOBALIZATION_NATIVE_DIR}")
-include_directories("${CLR_REPO_ROOT_DIR}/src/libraries/Native/Unix/Common")
+set(STATIC_LIBS_ONLY 1)
 
-add_subdirectory(${GLOBALIZATION_NATIVE_DIR} System.Globalization.Native)
+if(CLR_CMAKE_TARGET_WIN32)
+    set(GLOBALIZATION_NATIVE_DIR ${CLR_REPO_ROOT_DIR}/src/libraries/Native/Unix/System.Globalization.Native)
+    set(COMPRESSION_NATIVE_DIR ${CLR_REPO_ROOT_DIR}/src/libraries/Native/Windows/clrcompression)
 
+    include_directories("${GLOBALIZATION_NATIVE_DIR}")
+    include_directories("${COMPRESSION_NATIVE_DIR}")
+
+    include_directories("${CLR_REPO_ROOT_DIR}/src/libraries/Native/Unix/Common")
+
+    add_subdirectory(${COMPRESSION_NATIVE_DIR} clrcompression)
```

Comment:
Adding this sub directory should be responsible for adding the needed include paths no? I thought that was the CMake pattern - directories with a CMakelists.txt add the needed requirements to build them.

Reply:
"libraries-native" folder is gone now, so I consider this resolved.